docs/download.md

```diff
@@ -4,8 +4,40 @@ title: Download
 nav_order: 3
 ---
 
+[RAPIDS Accelerator For Apache Spark](https://github.com/NVIDIA/spark-rapids) provides a set of plugins for Apache Spark that leverage GPUs to accelerate Dataframe and SQL processing. The accelerator is built upon the [RAPIDS cuDF project](https://github.com/rapidsai/cudf) and [UCX](https://github.com/openucx/ucx/).
```

Comment:
The paragraphs should ideally be wrapped at a reasonable character length (e.g.: 100) because:
- it makes it easier to read when viewing it in a terminal when browsing the repository directly
- it makes it easier to target comments in documentation PRs.

Reply:
Done in latest commit. Feel free to let me know if it looks good.

Reply:
The lines are still very long.  For example, this paragraph is all one line in the file: https://github.com/NVIDIA/spark-rapids/pull/1894/files#diff-29528fb31258a56dda5f112cb5780226ebbf8016cadbdeb4948ad84b89ccf533R13

Reply:
Done.thx